src/main/java/com/okta/sdk/framework/PagedResults.java

```diff
@@ -19,7 +19,9 @@ public boolean isFirstPage() {
     }
 
     public String getNextUrl() {
-        return apiResponse.getLinks().get("next").getHref();
+		if (apiResponse.getLinks().get("next")!=null)
```

Comment:
@roadmixer88 I'm cleaning up the PR (and thanks again for submitting this)

If still interested in this can you avoid the duplicate call to `apiResponse.getLinks().get("next")` ? 
#hacktoberfest